docs/development.md

```diff
@@ -246,7 +253,7 @@ If you are testing against a production server (staging server or live server),
 1. Edit `src/e2e/resources/test.properties` as instructed is in its comments.
    * In particular, you will need legitimate Google accounts to be used for testing.
 
-1. Run the full test suite or any subset of it as how you would have done it in dev server.
+1. Login manually to TEAMMATES on the browser used for testing before running any E2E test one at a time. This is required as Google does not allow login by automated software.
```

Comment:
I still don't think "one at a time" is a necessary condition. If the reason is because of the tests that are injected with real accounts, then the solution is to make those tests sequential. Reference: <https://github.com/TEAMMATES/teammates/blob/9c60e07c260c3f4fd1b268d627f0cd4b9cda24da/src/test/testng-ci.xml>, take a look at the `sequential-ui-tests` part.

Reply:
It is actually because of this issue where we cannot open two instance of ChromeDriver with the same profile:
https://stackoverflow.com/questions/34727328/run-multiple-chrome-profiles-instances-for-automation-task

And we cannot login without using the profile data (or I have not found a viable solution).

I tried running all e2e tests sequentially but because we do not close the windows of test that fail, the profile becomes locked after one test fails and all subsequent test also fail. I think this can be changed though if we decide that running all the test sequentially for production server e2e is ok.

Firefox is ok.

Reply:
For now I have written one at a time for chrome, we can change this at a later date after some testing with the sequential runs? 

Might be good to add a new command for running sequentially or some flag if it works as well.